app/assets/javascripts/api/plots.js

```diff
@@ -21,31 +21,81 @@ export const plotIndexUrl = (queryParams) => {
   return path;
 };
 
-export const createPlot = (manifestId, plot) =>
-  fetch(Routes.manifests_plots_create_path(PROJECT_NAME, manifestId), {
+export const createPlot = (plot) =>
+  fetch(Routes.manifests_plots_create_path(PROJECT_NAME, plot.manifestId), {
     credentials: 'same-origin',
     method: 'POST',
     headers: headers('json', 'csrf'),
-    body: JSON.stringify(plot)
+    body: JSON.stringify(plotToJson(plot))
   })
     .then(checkStatus)
     .then(parseJSON)
+    .then(plotFromJson)
 
-export const destroyPlot = (manifestId, plotId) =>
-  fetch(Routes.manifests_plots_destroy_path(PROJECT_NAME, manifestId, plotId), {
+export const destroyPlot = (plot) =>
+  fetch(Routes.manifests_plots_destroy_path(PROJECT_NAME, plot.manifestId, plot.id), {
```

Comment:
These routes don't need the manifest id, they should just be plots_destroy_path, etc.

Reply:
are you saying the route needs to be changed?
delete ':project_name/manifests/:manifest_id/plots/destroy/:id'=> 'plots#destroy', as: :manifests_plots_destroy

Reply:
yeah, the route could just be like :project_name/plots/destroy/:id, since you don't need to know what manifest it is associated with